orchestrator/graphql/autoregistration.py

```diff
@@ -81,22 +80,22 @@ def create_block_strawberry_type(
     return strawberry_wrapper(new_type)
 
 
-def create_strawberry_enums(model: Type[DomainModel], strawberry_enums: EnumDict) -> EnumDict:
-    enums = {
-        key: field
-        for key, field in model._non_product_block_fields_.items()
-        if is_enum(field) and is_not_strawberry_enum(key, strawberry_enums)
-    }
-    return strawberry_enums | {key: create_strawberry_enum(field) for key, field in enums.items()}
-
-
 def add_class_to_strawberry(
     model_name: str,
     model: Type[DomainModel],
     strawberry_models: StrawberryModelType,
     strawberry_enums: EnumDict,
-    with_interface: bool = False,
+    interface: Any = None,
 ) -> None:
+    def create_subscription_strawberry_type(strawberry_name: str, model: Type[DomainModel]) -> Type[interface]:
+        base_type = type(strawberry_name, (interface,), {})
```

Comment:
As codecov mentions these lines probably aren't hit by a unittest. When `interface` has its default value of `None` this would raise a TypeError


```
In [18]: base = type('bla', (None,), {})
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [18], in <cell line: 1>()
----> 1 base = type('bla', (None,), {})

```

Reply:
This never gets called without a interface